source/includes/_portfolio.md

```diff
@@ -6,7 +6,7 @@ Portfolio data needs to be submitted per indicator, and requires many requests t
 
 `/api/v0/entities/{entity_id}/responses/{response_id}/values/{variable}`
 
-A full list of variables, along with their description, data type, and accepted values can be found in the Excel version of the [Real Estate assessment](https://gresb-prd-public.s3.amazonaws.com/2023/data-partners/2023-real-estate-variables.xlsx). See [introduction](#introduction) for the Postman collection.
+A full list of variables, along with their description, data type, and accepted values can be found in the Excel version of the [Real Estate assessment](https://gresb-prd-public.s3.amazonaws.com/2024/data_partners/2024_real_estate_variables.xlsx). See [introduction](#introduction) for the Postman collection.
```

Comment:
We decided to put this one - all valuable resources in general - in the introduction as well right?